bpf/cpu/cpu.bpf.c

```diff
@@ -72,6 +72,7 @@ _Static_assert(1 << MAX_BINARY_SEARCH_DEPTH >= MAX_UNWIND_TABLE_SIZE, "unwind ta
 #define REQUEST_UNWIND_INFORMATION (1ULL << 63)
 #define REQUEST_PROCESS_MAPPINGS (1ULL << 62)
 #define REQUEST_REFRESH_PROCINFO (1ULL << 61)
```

Comment:
I don't think we need another event type, `REQUEST_PROCESS_MAPPINGS` is sent when the stack is aggregated. We can change its name perhaps to make sure our intent here it's clear.

There should not be any drawbacks, as walking stacks, even deep ones, take less than a millisecond in our tests.